dynamo/plot/preprocess.py

```diff
@@ -743,15 +725,7 @@ def feature_genes(
 
     neg_disp_table = table.iloc[~valid_ind, :]
 
-    if mode == "dispersion":
-        ax = plt.scatter(
-            neg_disp_table["mean_expression"],
-            neg_disp_table["dispersion_empirical"],
-            s=3,
-            alpha=0.5,
-            color="xkcd:grey",
-        )
-    elif mode == "SVR":
+    if "_dispersion" in mode:
         ax = plt.scatter(
```

Comment:
"_dispersion" seems to be the only mode we support in this function. Adding "else" statements to avoid errors and allow more modes in the future may be a good idea. Alternatively, we can remove all the `if "_dispersion" in mode` statements, and rename the function to something like `feature_genes_dispersion`.

Reply:
There is 'gini' mode as well, but I am not sure it is also supposed to be plotted..